data/devhabib429.json

```
@@ -65,5 +65,5 @@
       "url": "https://ko-fi.com/devhabib429",
       "icon": "SiBuymeacoffee"
     }
-  ]
+  ],
```

```suggestion
  ]
```
Please remove trailing comma